src/test/java/com/fasterxml/jackson/databind/deser/creators/AnySetterForCreator562Test.java

```diff
@@ -0,0 +1,180 @@
+            this.a = a;
+            anySetterNode = leftovers;
+        }
+    }
+
+    static class MultipleAny562
+    {
+        @JsonCreator
+        public MultipleAny562(@JsonProperty("a") String a,
+            @JsonAnySetter Map<String, Object> leftovers,
+            @JsonAnySetter Map<String, Object> leftovers2) {
+            throw new Error("Should never get here!");
+        }
+    }
+
+    static class PojoWithDisabled
+    {
+        String a;
+        Map<String,Object> stuff;
+
+        @JsonCreator
+        public PojoWithDisabled(@JsonProperty("a") String a,
+            @JsonAnySetter(enabled = false) Map<String, Object> leftovers
+        ) {
+            this.a = a;
+            stuff = leftovers;
+        }
+    }
+
+    private final ObjectMapper MAPPER = newJsonMapper();
+
+    @Test
+    public void mapAnySetterViaCreator562() throws Exception
+    {
+        Map<String, Object> expected = new HashMap<>();
+        expected.put("b", Integer.valueOf(42));
+        expected.put("c", Integer.valueOf(111));
+
+        POJO562 pojo = MAPPER.readValue(a2q(
+                "{'a':'value', 'b':42, 'c': 111}"
+                ),
+                POJO562.class);
+
+        assertEquals("value", pojo.a);
+        assertEquals(expected, pojo.stuff);
+
+        // Should also initialize any-setter-Map even if no contents
+        pojo = MAPPER.readValue(a2q("{'a':'value2'}"), POJO562.class);
+        assertEquals("value2", pojo.a);
+        assertEquals(new HashMap<>(), pojo.stuff);
+    }
+
+    @Test
+    public void testNodeAnySetterViaCreator562() throws Exception
+    {
+        PojoWithNodeAnySetter pojo = MAPPER.readValue(
+                a2q("{'a':'value', 'b':42, 'c': 111}"),
+                PojoWithNodeAnySetter.class);
+
+        assertEquals("value", pojo.a);
+        assertEquals(a2q("{'c':111,'b':42}"), pojo.anySetterNode + "");
+    }
+
+    @Test
+    public void testAnyMapWithNullCreatorProp() throws Exception
+    {
+        ObjectMapper failOnNullMapper = jsonMapperBuilder()
+            .enable(DeserializationFeature.FAIL_ON_NULL_CREATOR_PROPERTIES).build();
+
+        // 08-Jun-2024, tatu: Should be fine as we get empty Map for "no any setters"
+        POJO562 value = failOnNullMapper.readValue(a2q("{'a':'value'}"), POJO562.class);
+        assertEquals(Collections.emptyMap(), value.stuff);
+    }
+
+    @Test
+    public void testAnyMapWithMissingCreatorProp() throws Exception
+    {
+        ObjectMapper failOnMissingMapper = jsonMapperBuilder()
+            .enable(DeserializationFeature.FAIL_ON_MISSING_CREATOR_PROPERTIES).build();
+
+        // Actually missing (no any props encountered)
+        try {
+            failOnMissingMapper.readValue(a2q("{'a':'value'}"), POJO562.class);
+            fail("Should not pass");
+        } catch (MismatchedInputException e) {
+            verifyException(e, "Missing creator property ''");
+        }
+
+        // But should NOT fail if we did find at least one...
```

Comment:
This fails now; check needs to consider that `Any` property is not empty (that is, had at least one value).
Not sure where the best place to check is -- also, how should semantics work for `JsonNode`?

Reply:
Fixed.